R/run.R

```diff
@@ -834,9 +871,15 @@ CmdStanMCMCProcs <- R6::R6Class(
           if (state < 3 && grepl("Iteration:", line, perl = TRUE)) {
             state <- 3 # 3 =  warmup
             next_state <- 3
+            private$iter_done_[[id]] = 0
+          }
+          if (grepl("Iteration:", line, perl = TRUE)) {
+            regex_match = regexpr('[0-9]+\\s+/',line)
+            private$iter_done_[[id]] = as.numeric(substr(line,regex_match,regex_match+attr(regex_match,'match.length')-3))
+            ignore_line = TRUE
```

Comment:
```suggestion
            regex_match <- regexpr('[0-9]+\\s+/',line)
            private$iter_done_[[id]] <- as.numeric(substr(line, regex_match, regex_match + attr(regex_match, 'match.length') - 3))
            ignore_line <- TRUE
```

Also same comment as above about splitting up the long line